src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/WamAdapters.cs

```diff
@@ -206,6 +211,19 @@ private enum ResponseStatus
             return authParams;
         }
 
+        private static void DecorateExceptionWithRuntimeErrorProperties(MsalException exception, Error runtimeError)
+        {            
+            var result = new Dictionary<string, string>()
+            {
+                { "RuntimeContext", runtimeError.Context },
+                { "RuntimeTag", runtimeError.Tag.ToString(CultureInfo.InvariantCulture)},
```

Comment:
should we return `0x{Tag:x}` so it matches the runtime tag value? if that would be helpful for msalruntime team to debug, not sure if they are converting the string we return to the tag value when they debug issues